nemo/collections/nlp/modules/common/sequence_token_classifier.py

```diff
@@ -34,7 +34,7 @@ def input_types(self) -> Optional[Dict[str, NeuralType]]:
     def output_types(self) -> Optional[Dict[str, NeuralType]]:
         return {
             "intent_logits": NeuralType(('B', 'D'), LogitsType()),
-            "slot_logits": NeuralType(('B', 'T', 'D'), LogitsType()),
+            "slot_logits": NeuralType(('B', 'T', 'C'), LogitsType()),
```

Comment:
Why is it 'C' but intent_logits is 'D'?

Reply:
Actually C and D resolve to the same thing. Some people say Channel, others Dimension

Reply:
I again copied it from you and Evelina code, which is not consistent. But Oleksii explained that it is basically the same, so I changed it to be 'D' everywhere.